cheroot/makefile.py

```diff
@@ -36,13 +36,22 @@ def write(self, b):
             self._flush_unlocked()
             return len(b)
 
+    def _safe_call(self, is_reader, call, *args, **kwargs):  # noqa: C901
```

Comment:
Are you sure it's C901? It doesn't look like it's complex.

Reply:
Oh, right, sorry, I just copied the method signature from `pyopenssl.SSLFileobjectMixin` including the comment :)